### PR TITLE
Update Read.me

### PR DESCRIPTION
**Pull request by sanchaymittal**
Better Documentation Better Understanding
Please label this as Hacktoberfest

### CODE REVIEW

**Comment by sanchaymittal** [comment]
Issue of bad representation have been resolved.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+* DFS
+* Bit manipulation
+* Computational geometry
+* Data structures
+* Divide conquer
+* Dynamic programming
+* Graph algorithms 
+* Greedy algorithms
+* Mathematical algorithms
+* Randomized algorithms
```
Comment (matthewsamuel95) [inline README.md]: Randomized algorithms are not here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+**Table of Contents**
+
+
+* Breadth First Traversal
+* Backtracking
+* DFS
+* Bit manipulation
+* Computational geometry
+* Data structures
+* Divide conquer
```
Comment (matthewsamuel95) [inline README.md]: Divide / DS algorithms are not here

File: README.md (hunk 1 above)
Comment (sanchaymittal) [reply README.md]: Ohh, i haven't noticed

Comment (sanchaymittal) [reply README.md]: just a sec i will add these algos

Comment (matthewsamuel95) [reply README.md]: Also links would be nice

Comment (sanchaymittal) [reply README.md]: i will create it as issue just assign it to me.

File: README.md (hunk 2 above)
Comment (sanchaymittal) [reply README.md]: Divide conquer is concept used in quick sort ,binary search etc.

**Pull request closed without merging**